.gitignore

```diff
@@ -28,11 +27,10 @@ Session.vim
 *.save
 node_modules/
 stacks-node-follower/Config.toml
-configurations/*/Config.toml
-configurations/private-testnet/*.toml
```

Comment:
I wouldn't address private-testnet files in this PR, it's out of scope

Reply:
ok. added them back in again.